### ISSUE TITLE
Notes confusing to screenreader users

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/h2o#657]
Expanded notes are read before the text they annotate (I believe because of the DOM order, though I haven't looked at the source). However, they are not described as a note: there is no way to distinguish them from the casebook text. And, they are not associated with the text they annotate (except by proximity). That UX needs some designing.... hidden text reading "Note from author:" before the note next would help, and "end note from author" at the end, as a first pass. I'll have a think on whether there is a better technique, and whether something like `aria-describedby` could be used to programmatically associate the note with the text that it annotates.

**Comment by rebeccacremona** [comment]
I see no evidence that I ever worked on notes on prod... this is a long-standing issue, and I seem to have gotten stuck in the research phase. 

Since there don't seem to be many notes at present, maybe not the highest priority. And since this isn't yet fixed on prod, certainly not a blocker to deploying the Vue code!

**Comment by leppert** [comment]
https://github.com/harvard-lil/h2o/commit/dc985e53261f823b83ba0f2693af72c9c7194b52 changes the order such that the selected text should be read first
